content/docs/main/agentgateway/llm/providers/bedrock.md

```diff
@@ -0,0 +1,154 @@
+             auth:
+               type: Secret
+               secretRef:
+                 name: bedrock-secret
+   EOF
+   ```
+
+   {{% reuse "docs/snippets/review-table.md" %}} For more information or other providers, see the [API reference](/docs/reference/api/#aibackend).
+
+   | Setting     | Description |
+   |-------------|-------------|
+   | `type`      | Set to `AI` to configure this Backend for an AI provider. |
+   | `ai`        | Define the AI backend configuration. The example uses Amazon Bedrock (`spec.ai.llm.provider.bedrock`). |
+   | `model`     | The model to use to generate responses. In this example, you use the `amazon.titan-text-lite-v1` model. For more models, see the [AWS Bedrock docs](https://docs.aws.amazon.com/bedrock/latest/userguide/models-supported.html). |
+   | `region`    | The AWS region where your Bedrock model is deployed. Multiple regions are not supported. |
+   | `auth` | Provide the credentials to use to access the Amazon Bedrock API. The example refers to the secret that you previously created. To use IRSA, omit the `auth` settings.|
+
+4. Create an HTTPRoute resource to route requests through your agentgateway Gateway to the Bedrock Backend. Note that {{< reuse "docs/snippets/kgateway.md" >}} automatically rewrites the endpoint that you set up (such as `/bedrock`) to the appropriate chat completion endpoint of the LLM provider for you, based on the LLM provider that you set up in the Backend resource.
+   ```yaml
+   kubectl apply -f- <<EOF                                             
+   apiVersion: gateway.networking.k8s.io/v1
+   kind: HTTPRoute
+   metadata:       
+     name: bedrock
+     namespace: {{< reuse "docs/snippets/namespace.md" >}}
+   spec:
+     parentRefs:
+       - name: agentgateway
+         namespace: {{< reuse "docs/snippets/namespace.md" >}}
+     rules:
+     - matches:
+       - path:
+           type: PathPrefix
+           value: /bedrock
+       backendRefs:
+       - name: bedrock
+         namespace: {{< reuse "docs/snippets/namespace.md" >}}
+         group: gateway.kgateway.dev
+         kind: Backend
+   EOF
+   ```
+5. Send a request to the LLM provider API. Verify that the request succeeds and that you get back a response from the chat completion API.
+
+   {{< tabs tabTotal="2" items="Cloud Provider LoadBalancer,Port-forward for local testing" >}}
+   {{% tab tabName="Cloud Provider LoadBalancer" %}}
+   ```sh
+   curl "$INGRESS_GW_ADDRESS:8080/bedrock" -H content-type:application/json -d '{
+       "messages": [
+         {
+           "role": "user",
+           "content": [
+             {
+               "text": "You are a cloud native solutions architect, skilled in explaining complex technical concepts such as API Gateway, microservices, LLM operations, kubernetes, and advanced networking patterns. Write me a 20-word pitch on why I should use an AI gateway in my Kubernetes cluster."
+             }
+           ]
+         }
+       ]
+     }' | jq
```

Comment:
I often need to provide the model also, it can be set to model="", but is often required for the request to go through